cobc/cconv.c

```diff
@@ -20,6 +20,9 @@
 */
 #include "cconv.h"
 
+#include <stdio.h>		/* for FILE (FIXME: move to cobc.h?) */
```

Comment:
here's that comment again :-)

OK, let's include that in cobc.h, but or now without adjusting other files.